src/python/WMCore/MicroService/Unified/MSRuleCleaner.py

```diff
@@ -108,8 +108,14 @@ def __init__(self, msConfig, logger=None):
         # Initialization of the 'cleaned' and 'archived' counters:
         self.wfCounters = {'cleaned': {},
                            'archived': 0}
+
+    def zeroCounters(self):
```

Comment:
Given that you have to sort that unit test out, let me say that `resetCounters` is a more natural name ;)

Reply:
Done 
And also unit test fixed